src/CarbonAware.WebApi/src/Models/EmissionsForecastDTO.cs

```diff
@@ -67,9 +67,30 @@ public static EmissionsForecastDTO FromEmissionsForecast(EmissionsForecast emiss
             DataStartAt = emissionsForecast.DataStartAt,
             DataEndAt = emissionsForecast.DataEndAt,
             WindowSize = (int)emissionsForecast.WindowSize.TotalMinutes,
-            OptimalDataPoint = EmissionsDataDTO.FromEmissionsData(emissionsForecast.OptimalDataPoint),
```

Comment:
_If we elect to not change the API contract:_ 

This line should change to properly grab the single value from core forecast model enumerable:

`OptimalDataPoint = EmissionsDataDTO.FromEmissionsData(emissionsForecast.OptimalDataPoints.FirstOrDefault()),`